dev/io.openliberty.org.apache.myfaces.4.0_fat/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=--init-script /var/folders/xl/j2vjtp1925395jm3l1jm5g5h0000gn/T/d146c9752a26f79b52047fb6dc6ed385d064e120494f96f08ca63a317c41f94c.gradle --init-script /var/folders/xl/j2vjtp1925395jm3l1jm5g5h0000gn/T/52cde0cfcf3e28b8b7510e992210d9614505e0911af0c190bd590d7158574963.gradle
```

Comment:
What is this file used for? Not sure I've seen it before :)

Reply:
Ooops accidentally opened this in VS Code which generated this file.